docs/GetStarted/buildRPC.md

```diff
@@ -132,8 +132,8 @@ The toolkit provides a reference implementation called the Sample Web UI to mana
         </figure>
 
 
-    !!! error "Troubleshooting"
-        Run into an issue? Try these [troubleshooting steps](../Reference/troubleshooting.md).
+        !!! error "Troubleshooting"
```

Comment:
I don't think you meant to put this inside of success?

![image](https://user-images.githubusercontent.com/22204562/194164844-a8de2da3-b6ef-4442-8c1b-7cd9ee1c8db4.png)